### PR TITLE
feat: add eslint-plugin-jquery

### PR DESCRIPTION
**Pull request by aaarichter**
https://github.com/dgraham/eslint-plugin-jquery

### CODE REVIEW

**Comment by dbushong** [comment]
I'm reading https://github.com/dgraham/eslint-plugin-jquery#eslint-plugin-jquery and I'm not sure - are any of these enabled by default?

Can you add a check or two in `examples/eslint`?  (Only has to be for say node8)

**Comment by aaarichter** [comment]
@dbushong I updated the commit with:
- adding the no-jquery eslint full ruleset
- adding test files to both `es5` and `node8` example folders

File: test/lint.test.js [hunk 1]
```diff
@@ -50,6 +50,10 @@ async function validate(filename, content) {
 }
 
 describe(`Linting using ESLint`, () => {
+  // const files = globby.sync(`examples/**/*.{mjs,js}`, {
+  //   ignore: ['examples/node_modules/**/*.*'],
+  // });
+
```
Comment (dbushong) [inline test/lint.test.js]: ?

Comment (aaarichter) [reply test/lint.test.js]: ups . removing

**Pull request closed without merging**